specparam/utils/data.py

```diff
@@ -10,6 +10,25 @@
 ###################################################################################################
 ###################################################################################################
 
+def get_freq_ind(freqs, freq):
```

Comment:
- What happens if there are `np.nan` values?
- What happens if there are more than one close value to `freq`? I suppose that the method pick the index of the first one. is that the expected behaviour?